Src/AutoFixture/DataAnnotations/IMemberTypeResolver.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace AutoFixture.DataAnnotations
+{
+    public interface IMemberTypeResolver
```

Comment:
@zvirja I had to introduce this interface to be able to test the Relay - but I found it is something that was not practiced before - injecting dependencies into the relays. Unfortunately, type FakeMamberInfo does not provide a way to mock a MemberType, and I am not sure if that is even possible for it to do. Maybe you can figure out a better way to achieve this testability without introducing this interface?